presto-main/src/main/java/com/facebook/presto/sql/Serialization.java

```diff
@@ -37,6 +37,9 @@
 
 public final class Serialization
 {
+    private static final char VARIABLE_TYPE_OPEN_BRACKET = '<';
```

Comment:
Instead of changing parentheses to bracket, can we change `SymbolAllocator` instead? We should handle the MagicalLiteral function calls in the `SymbolAllocator` specially so that we don't have have any '()' inside a symbol/variable. 

We also should consider escape '()' in name hint as well. I am more worried about somebody write a column name quoted with " and has a parenthesis inside.